itou/eligibility/models/common.py

```diff
@@ -70,6 +70,10 @@ def save(self, *args, **kwargs):
     def is_valid(self):
         return bool(self.expires_at and self.expires_at > timezone.now())
 
+    @property
+    def criteria_certification_required(self):
```

Comment:
Je ne sais pas trop comment on compte gérer le cas de l’API particulier indisponible / ne connaissant pas l’individu, mais il y a des chances qu’on ne bloque pas une embauche parce qu’on n’a pas réussi à certifier le critère ?
```suggestion
    def criteria_certification_available(self):
```

Reply:
En effet. J'aime bien ta proposition de nom.